Bento/Views/BentoReusableView.swift

```diff
@@ -20,6 +27,20 @@ extension BentoReusableView {
             containedView = nil
```

Comment:
Maybe we should release the component here? In terms of correctness if there's no `containedView` we shouldn't have a `component` as well.

Reply:
well, the incoming component is optional already so if it's `nil` the it will be set to `nil` 😛